### ISSUE TITLE
some basic testing of validate-bids CLI

### ISSUE THREAD

**Issue by yarikoptic** [dandi/dandi-cli#997]
looking at https://github.com/dandi/dandi-cli/pull/992/files#diff-4439db8365d1a65b4e704ee9824cbbaa9fa46527aa91dbe1e758c541bcb83affL24 - it seems that it is not covered by unittests at all (not sure if `validate` either).  

<details>
<summary>you can see how other click CLIs are validated in other tests:</summary> 

```shell
$> git grep CliRunner
dandi/cli/tests/test_command.py:from click.testing import CliRunner
dandi/cli/tests/test_command.py:    runner = CliRunner()
dandi/cli/tests/test_command.py:    runner = CliRunner()
dandi/cli/tests/test_digest.py:from click.testing import CliRunner
dandi/cli/tests/test_digest.py:    runner = CliRunner()
dandi/cli/tests/test_digest.py:    runner = CliRunner()
dandi/cli/tests/test_digest.py:    runner = CliRunner()
dandi/cli/tests/test_download.py:from click.testing import CliRunner
dandi/cli/tests/test_download.py:    r = CliRunner().invoke(download)
dandi/cli/tests/test_download.py:    r = CliRunner().invoke(download, ["--download", "all"])
dandi/cli/tests/test_download.py:    r = CliRunner().invoke(download, ["--download", "dandiset.yaml"])
dandi/cli/tests/test_download.py:    r = CliRunner().invoke(download, ["--download", "assets"])
dandi/cli/tests/test_download.py:    r = CliRunner().invoke(download, ["--download", "foo"], standalone_mode=False)
dandi/cli/tests/test_download.py:    runner = CliRunner()
dandi/cli/tests/test_download.py:    runner = CliRunner()
dandi/cli/tests/test_download.py:    r = CliRunner().invoke(
dandi/cli/tests/test_download.py:    r = CliRunner().invoke(
dandi/cli/tests/test_instances.py:from click.testing import CliRunner
dandi/cli/tests/test_instances.py:    r = CliRunner().invoke(instances, [])
dandi/cli/tests/test_ls.py:from click.testing import CliRunner
dandi/cli/tests/test_ls.py:    runner = CliRunner()
dandi/cli/tests/test_ls.py:    r = CliRunner().invoke(
dandi/cli/tests/test_ls.py:    r = CliRunner().invoke(
dandi/cli/tests/test_ls.py:    r = CliRunner().invoke(
dandi/cli/tests/test_ls.py:    runner = CliRunner()
dandi/tests/fixtures.py:from click.testing import CliRunner
dandi/tests/fixtures.py:    r = CliRunner().invoke(
dandi/tests/fixtures.py:    r = CliRunner().invoke(organize, ["-f", "move", "--dandiset-path", str(tmp_path)])
dandi/tests/test_organize.py:from click.testing import CliRunner
dandi/tests/test_organize.py:    r = CliRunner().invoke(organize, cmd)
dandi/tests/test_organize.py:    r = CliRunner().invoke(organize, cmd + ["--invalid", "warn"])
dandi/tests/test_organize.py:    r = CliRunner().invoke(organize, args)
dandi/tests/test_organize.py:    r = CliRunner().invoke(organize, cmd)
dandi/tests/test_organize.py:    r = CliRunner().invoke(organize, cmd)
```
</details>

**Comment by TheChymera** [comment]
Solved as of 59c8e60a17186b95ebbe4104f1ce3c8dfdb86845

**Comment by yarikoptic** [comment]
let's assume so
